recipes/geographiclib/all/conanfile.py

```diff
@@ -132,12 +128,19 @@ def package(self):
         ]:
             rmdir(self, os.path.join(os.path.join(self.package_folder, folder)))
         rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+        if not self.options.tools:
+            rmdir(self, os.path.join(self.package_folder, "sbin"))
+            bin_files = [it for it in os.listdir(os.path.join(self.package_folder, "bin")) if not it.endswith(".dll")]
+            for it in bin_files:
+                rm(self, it, os.path.join(self.package_folder, "bin"))
```

Comment:
Another possible is because is the past we usually added tools option for almost everything. But indeed need to re-check the initial commit.

Reply:
@SpaceIm if you don't know why this option is here (https://github.com/conan-io/conan-center-index/pull/4295), I'd say we should ask @zeeshancs07 why they need it, what they expect it to do, and if possible remove the option

Reply:
I guess the question is: would it be a breaking change if we removed this option, and always build and package the tools ?

Reply:
Do not need to remove, the upstream confirmed a way to avoid tools, so it's something officially supported: https://github.com/geographiclib/geographiclib/pull/27#issuecomment-2133322100